lms/envs/common.py

```diff
@@ -2870,3 +2870,8 @@
 TYPES_WITH_CHILD_PROBLEMS_LIST = [
     'library_content',
 ]
+
+# These are the problem types (by block_type) that an instructor can query over in the Instructor Query tab
+INSTRUCTOR_QUERY_PROBLEM_TYPES = [
+    'problem'
+]
```

Comment:
It's actually what JimC had not edx. In my mind, this makes it really confusing. If I'm trying to debug an issue and I want to see what types are supported I have to remember that this list is being overridden by another list in another repo. It would be easier to figure that out if this list was empty. Just my 2p.